lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -176,7 +176,7 @@ def get_virtual_facts(self):
                     virtual_facts['virtualization_type'] = 'RHEV'
                     found_virt = True
 
-        if product_name in ('VMware Virtual Platform', 'VMware7,1'):
+        if product_name and product_name.startswith(("VMware",)):
```

Comment:
FWIW the passed value doesn't need to be a tuple of strings if there's just one string.
```suggestion
        if product_name and product_name.startswith("VMware"):
```
would do the same but looks cleaner